app/invocation/invocation_timing_card.tsx

```diff
@@ -74,8 +74,14 @@ export default class InvocationTimingCardComponent extends React.Component<Props
   }
 
   getProfileFile(): build_event_stream.File | undefined {
+    const profileOption = this.props.model
+      .structuredCommandLine!.find((scl) => scl.commandLineLabel == "canonical")!
+      .sections.find((s) => s.sectionLabel == "command options")!
+      .optionList!.option.find((o) => o.optionName == "profile");
+    const profileName = profileOption ? profileOption.optionValue : "command.profile.gz";
```

Comment:
nit: avoid using non-null assertions

```suggestion
    const profileName = this.props.model
      .structuredCommandLine?.find((scl) => scl.commandLineLabel == "canonical")?
      .sections?.find((s) => s.sectionLabel == "command options")?
      .optionList?.option?.find((o) => o.optionName == "profile")?.optionValue ?? "command.profile.gz";
```

Reply:
Ohhh.. I do like this!